content/docs/examples/multicluster/iks-icp/index.md

```diff
@@ -0,0 +1,91 @@
+using the [Kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Set up two clusters
+
+1.  [Install One IBM Cloud Private clusters](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html).
+    __NOTE__: You can configure Pod CIDR ranges and service CIDR ranges by `network_cidr` and
+    `service_cluster_ip_range` in `cluster/config.yaml` for IBM Cloud Private.
+
+    {{< text plain >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  [Request One IBM Cloud Kubernetes Service clusters](https://console.bluemix.net/docs/containers/container_index.html).
+    __NOTE__: By default, when you have provisioned a IBM Cloud Kubernetes Service cluster, the CIDR is as below.
+
+    {{< text plain >}}
+    pod subnet CIDR: 172.30.0.0/16.
+    service subnet CIDR: 172.21.0.0/16.
+    {{< /text >}}
+
+## Configure pod communication across IBM Cloud Kubernetes Service & IBM Cloud Private
+
+Since these two clusters are in isolated network environments we need to set up VPN connection between them.
+
+1.  Set up strongSwan in IBM Cloud Kubernetes Service cluster.
+
+    1.  Set up helm in IBM Cloud Kubernetes Service by following [these instructions](https://console.bluemix.net/docs/containers/cs_integrations.html).
+
+    1.  Install strongSwan using helm chart by following [these instructions](https://console.bluemix.net/docs/containers/cs_vpn.html),Example configuration parameters from `config.yaml`:
+
+        {{< text plain >}}
+        ipsec.auto: add
+        remote.subnet: 10.0.0.0/24,10.1.0.0/16
+        {{< /text >}}
+
+    1.  Get the external IP of the `vpn-strongswan` service.
+
+        {{< text bash >}}
+        $ kubectl get svc vpn-strongswan
+        {{< /text >}}
+
+1.  Set up strongSwan in IBM Cloud Private.
+
+    1.  Complete the strongSwan workarounds for IBM Cloud Private by following [these instructions](https://www.ibm.com/support/knowledgecenter/SS2L37_2.1.0.3/cam_strongswan.html).
+
+    1.  Install the strongSwan from the catalog in the management console by following [these instructions](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/app_center/create_release.html),example configuration parameters:
+
+        {{< text plain >}}
+        Namespace: default
+        Operation at startup: start
+        Local subnets: 10.0.0.0/24,10.1.0.0/16
+        Remote gateway: Public IP of IKS vpn-strongswan service that you get earlier
+        Remote subnets: 172.30.0.0/16,172.21.0.0/16
+        Privileged authority for VPN pod: checked
+        {{< /text >}}
+
+    1.  Verify that IBM Cloud Private can connect to IBM Cloud Kubernetes Service by running the following command against cluster on the IBM Cloud Kubernetes Service:
```

Comment:
against cluster on the IBM Cloud Kubernetes Service -> on the IBM Cloud Kubernetes Service cluster